misc/python/materialize/checks/mzcompose_actions.py

```diff
@@ -42,12 +42,22 @@ def execute(self, e: Executor) -> None:
 
         image = f"materialize/materialized:{self.tag}" if self.tag is not None else None
         print(f"Starting Mz using image {image}")
+
+        # Due to a bug, this parameter needs to be set to False for older versions
```

Comment:
I'd prefer to link the bugs for reference

Reply:
There is no bug unfortunately just a fix that references a slack discussion. Will try to come up with a better comment, thanks for flagging.

Reply:
All comments updated to reference the PR